API-Server/src/main/java/swm/hkcc/LGTM/app/modules/mission/domain/Mission.java

```diff
@@ -4,8 +4,9 @@
 import lombok.*;
 import swm.hkcc.LGTM.app.global.entity.BaseEntity;
```

Comment:
Oracle의 Java package convention에 따르면, 패키지명은 전부 소문자로 쓰라고 되어 있어 LGTM을 소문자로 바꾸면 어떨까 합니다.
"Package names are written in all lower case to avoid conflict with the names of classes or interfaces."
[https://docs.oracle.com/javase/tutorial/java/package/namingpkgs.html](url)

이렇게 바꾸면 SonarQube의 검증 결과에서도 나오지 않을 것입니다.

Reply:
넵! 이부분은 팀원들과 논의한 뒤 한번에 수정하겠습니다!